fastlane/lib/assets/DefaultFastfileTemplate.swift

```diff
@@ -24,27 +24,26 @@ class Fastfile: LaneFile {
 
     func testLane() {
         desc("Runs all the tests")
-        scan()
+        run_tests()
     }
 
     func betaLane() {
         desc("Submit a new Beta Build to Apple TestFlight. This will also make sure the profile is up to date")
 
-        match(gitUrl: "gitUrl", appIdentifier: [appIdentifier], username: appleID)
-        // Build your app - more options available
-        gym([[SCHEME]])
-        pilot(username: appleID)
+        // sync_code_signing(gitUrl: "gitUrl", appIdentifier: [appIdentifier], username: appleID)
```

Comment:
Commented out `sync_code_signing` (aka `match`) to match the Ruby `Fastfile` template. If this is in error, I'm happy to revert!

Reply:
need to update the code that substitutes these values in fastlane during initialization `[SCHEME]`

Reply:
I'm not sure I follow. The code doesn't match on the action name, so I'd assume there would be no changes to be made there.

Reply:
We look for these tokens during initialization and substitute values, check out https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/setup/setup_ios.rb#L301
That way, when you get the fastfile back, it has all the correct values filled in that were specified during `init`.

Reply:
Ok, ignore me, I took a closer look and you're got it 💯 correct.

Reply:
👍